File: Treinamento/Java/Estrutura_de_decisão/Questão04.java
import java.util.Scanner;
public class Questão04 {
    public static void main(String[] args) {
        Scanner s;
        s = new Scanner(System.in);

        System.out.print("Digite a primeira nota: ");
        double n1 = s.nextDouble();

        System.out.print("Digite a segunda nota: ");
        double n2 = s.nextDouble();
    
        double media = (n1 + n2) / 2.0;
        System.out.printf("Média do aluno: %.2f\n", media);

        if (media < 7) {
            System.out.println("Você foi reprovado");
        } else if (media >= 7 && media < 10) {
            System.out.println("Você foi aprovado");
        } else {
            System.out.println("Você foi aprovado com distinção");
        }
    }
}        

File: Treinamento/C/Estrutura_de_decisão/Questão04.c
#include <stdio.h>

int main() {
    double n1;
    double n2;
    
    printf("Digite a primeira nota: ");
    scanf("%lf", &n1);

    printf("Digite a segunda nota: ");
    scanf("%lf", &n2);

    double media = (n1 + n2) / 2;
    printf("Média: %lf\n", media);
    
    if (media < 7) {
        printf("Reprovado\n");
    } else if (media <= 9.9) {
        printf("Aprovado\n");
    } else {
        printf("Aprovado com distinção\n");
    }
}
